# Results DNA methylation analysis

DNA was isolated from ~ 50 mg of EAT and SAT samples using standard proteinase K digestion and phenol-chloroform extraction method. DNA quality and quantity were analyzed using the NanoDrop 8000 UV–Vis spectrophotometer (Thermo Scientific; Waltham, MA). DNA methylation status of ABCA1 and ABCG1 CpG-sites was assessed by pyrosequencing using PyroMark Q24 (Qiagen) according to the manufacturer’s instructions. Biotin-labeled single-stranded amplicons were retrieved and subjected to pyrosequencing with use of 0.3 μM sequencing primer, according to the manufacturer’s protocol. The percent of methylation for each of the CpG sites within the target sequence was calculated using PyroQ CpG Software (Qiagen). Non-CpG cytosine residues were used as built-in controls to verify bisulfite conversion. Each sample was tested in two replicates and their average was used in the statistical analysis.

Genes and primers for pyrosequencing

Gene Genomic position of CpG sites* (GCRh37/hg19) Primer sequence ABCA1*

chr9:107,690,762; chr9:107,690,770 (cg14019050); chr9:107,690,773; chr9:107,690,791; chr9:107,690,797 
5’-AACAAATTCCACTAATACCCTTAACT-3’ 5’-biotin-GGGTGGAGGGTATAGTAGGT-3’ Seq 5’-AACAAATTCCACTAATACCCTTAACT-3’

ABCG1

chr21:43,642,336; chr21:43,642,354;chr21:43,642,367 (cg27243685) 5’-TGAGTTTAGGAGGTTAAGGAGAAATT-3’ 5’-biotin-CAAATAAACCAACAACAAAACAATAC-3’ Seq 5’-TGAGTTTAGGAGGTTAAGGA-3’

chr21:43,656,587 (cg06500161); chr21:43,656,590 5'-GTAAGGTTTGGGGTTATTTTAGTGG-3’ 5’-biotin-AAAACAAACCCTTAAATCTCTTTACAT-3’ Seq 5’-GAGATTAGGGTTTTTTTTAGATA-3’

*For ABCA1 promoter region average methylation level of promoter 5 CpG-sites was calculated.

Methylation_ABCA1_ABCG1.xlsx -- DNA methylation status of ABCA1 and ABCG1 CpG-sites
Pyro_Results (folder) -- Examples of pyrograms for each locus
